src/utils/queries.ts

```diff
@@ -34,7 +34,7 @@ export const constructCancelTxQuery = (txConfig: ITxConfig, newGasPrice: number)
   return queryString.stringify({
     ...unfinishedSpeedUpTxQueryParams,
     to: unfinishedSpeedUpTxQueryParams.from,
-    data: '0x0',
+    data: '0x',
```

Comment:
Did this break somewhere?

Reply:
I was playing around with it and couldn't cancel transactions without this change. Because the gas calculation would be off.

Reply:
I reached out to you on Slack to verify that it was an issue.